packages/@aws-cdk/aws-s3-deployment/lib/bucket-deployment.ts

```diff
@@ -107,6 +108,13 @@ export interface BucketDeploymentProps {
    */
   readonly memoryLimit?: number;
 
+  /**
+   *  Mount an EFS file system. Enable this if your assets are large and you encounter disk space errors.
```

Comment:
Add a note that specifying this option requires to also specify a `vpc` (you have it on the other side, which is good, but it's missing here).

Reply:
Updated as below
```
  /**
   *  Mount an EFS file system. Enable this if your assets are large and you encounter disk space errors.
   *  Enabling this option will require a VPC to be specified.
   *
   * @default - No EFS. Lambda has access only to 512MB of disk space.
   */
```